en/services/mission.md

```diff
@@ -379,6 +379,51 @@ Note:
 - [timeouts](#timeout) are not considered errors.
 - Out-of-sequence messages in mission upload/download are recoverable, and are not treated as errors.
 
+## Executing Missions
+
+Following mission upload, or vehicle restart, or mission reset, all jump counter loops are set to their initial values.
+Unless a mission has just been uploaded that explicitly sets the current mission item, the current mission item is set to 1.
```

Comment:
@peterbarker Yes I do. Thanks for clarification. There is a further exception here that if you are flying the mission item is not reset. That allows uploading a new mission and it continuing on the same mission item.
```suggestion
The current mission item is normally reset to one on mission reset or vehicle restart, or on mission upload when the vehicle is landed. The current mission item may retain its current value if a mission is uploaded in-air or it may be set to the mission item that has `current` set in a mission item (this field is ignored in ArduPilot but respected by PX4).
```